recipe/meta.yaml

```diff
@@ -106,7 +115,11 @@ outputs:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
```

Comment:
This needs to be added to the top level build requirements in addition to all the outputs.

Reply:
There is a way to avoid duplicating dependencies. I think you have to name the top level package the same as the "first package".

I'm not sure if this is a feature or bug, but I use it in:
https://github.com/conda-forge/opencv-feedstock/blob/master/recipe/meta.yaml#L56

Would it be acceptable to use it here too?